contrib/awslambda/python/src/python/pants/contrib/awslambda/python/BUILD

```diff
@@ -5,7 +5,6 @@ contrib_plugin(
   name='plugin',
   dependencies=[
     'contrib/awslambda/python/src/python/pants/contrib/awslambda/python/rules',
-    'contrib/awslambda/python/src/python/pants/contrib/awslambda/python/subsystems',
```

Comment:
The below AWS subsystem changes are needed so that we don't get an error with `lambdex` being declared twice as an options scope when running `release.sh`.